### ISSUE TITLE
MLCad-like model list

### ISSUE THREAD

**Issue by mjhorvath** [leozide/leocad#366]
Would it be possible to add a model/parts list like the one used in MLCad or before it LEdit?

Item #18 in this image:

http://www.holly-wood.it/images/mlcad/easyrot/ml_easyrot-screen8-en.gif

It doesn't have to be enabled by default. Just so I can access it when I need it.

Thanks.

**Comment by leozide** [comment]
We have the timeline widget:

![image](https://user-images.githubusercontent.com/9685066/60903672-a0199980-a226-11e9-81cf-63dbcc95a022.png)

What features are you missing from the MLCad list?

**Comment by mjhorvath** [comment]
Mainly being able to see the position, rotation matrix, color, etc. all in one place. Occasionally I will misposition or misrotate parts by small decimal values, and I can see that the values aren't rounded to the nearest integer or multiple of 20 or 8. It is also useful sometimes to see the filename of a part as well as its description.

Basically, it's sometimes nice to have all this info in one panel instead of spread across several panels.

**Comment by Symbian9** [comment]
> Basically, it's nice to have all this info in one panel instead of spread across several panels.

It would be cool add such panel and few additional toolbars from LDPub3D.

> Of course not everyone needs this.

For that reason, it would be good to add "Developer mode" into "Preferences" dialog for enable/disable additional toolbars.

**Comment by mjhorvath** [comment]
Yeah, an "Expert" mode and a "Novice" mode might be a good idea eventually.

**Comment by nathaneltitane** [comment]
> Yeah, an "Expert" mode and a "Novice" mode might be a good idea eventually.

or more widgets to tailor the expeience to the needs of what'S being created, edited?

**Comment by mjhorvath** [comment]
Yes. That too.

**Comment by gargan26** [comment]
Not being able to edit the underlying LDraw file data at any time is what really keeps me from adopting LeoCAD over MLCad.

**Comment by leozide** [comment]
I don't mind adding position and rotation to the tooltip but I don't like the idea of showing all of that information to the user by default.
